Doesn't compile from fresh install (doesn't recognize iobank0_hw)

I clone this repo, launch ./build.sh, and this is what i get:

```
[ 20%] Building CXX object CMakeFiles/picorvd.dir/src/Console.cpp.o
[ 21%] Building CXX object CMakeFiles/picorvd.dir/src/GDBServer.cpp.o
[ 22%] Building CXX object CMakeFiles/picorvd.dir/src/utils.cpp.o
/build/picorvd/src/PicoSWIO.cpp: In member function 'void PicoSWIO::reset(int)':
/build/picorvd/src/PicoSWIO.cpp:61:3: error: 'iobank0_hw' was not declared in this scope; did you mean 'io_bank0_hw'?
   61 |   iobank0_hw->io[pin].ctrl = GPIO_FUNC_SIO << IO_BANK0_GPIO0_CTRL_FUNCSEL_LSB;
      |   ^~~~~~~~~~
      |   io_bank0_hw
make[2]: *** [CMakeFiles/picorvd.dir/build.make:94: CMakeFiles/picorvd.dir/src/PicoSWIO.cpp.o] Error 1
```

Regardless of whether I offer it the latest Pico SDK repo via `PICO_SDK_PATH` or let it download its own copy of the SDK, the result is the same.

When I look into the SDK:
```
$ git grep -i 'iobank0_hw'
src/rp2040/hardware_structs/include/hardware/structs/iobank0.h:#define iobank0_hw io_bank0_hw
src/rp2350/hardware_structs/include/hardware/structs/iobank0.h:#define iobank0_hw io_bank0_hw
```

Do you have any ideas or advices?

--- a/pico_sdk_import.cmake
+++ b/pico_sdk_import.cmake
@@ -34,14 +34,14 @@ if (NOT PICO_SDK_PATH)
             FetchContent_Declare(
                     pico_sdk
                     GIT_REPOSITORY https://github.com/raspberrypi/pico-sdk
-                    GIT_TAG master
+                    GIT_TAG 1.5.1
                     GIT_SUBMODULES_RECURSE FALSE
             )
         else ()
             FetchContent_Declare(
                     pico_sdk
                     GIT_REPOSITORY https://github.com/raspberrypi/pico-sdk
-                    GIT_TAG master
+                    GIT_TAG 1.5.1
             )
         endif ()
 

